EmaPajic/assignment2/assignment2_Q1.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+@author: EmaPajic
+"""
+import unittest
+
+"""If we assume that we can have a pointer to the parent"""
+
+class TreeNode:
+    def __init__(self, val, left = None, right = None, parent = None):
+        self.val = val
+        self.left = left
+        self.right = left
+        self.parent = parent
+        
+"""We just go through pointers to parent until we reach None"""
```

Comment:
Can you put the docstring below the function definition line? E.g.

def find_ancestors(node):
 """Comment"""

Reply:
updated :)